### PR TITLE
Icon: Add compose new message icon

### PR DESCRIPTION
**Pull request by jcpark89**
Add new Icon into Gestalt

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit ad98882cb5b98d1f4bf27e4f2069f04cd8807449

https://deploy-preview-358--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :file_folder: Bundlesize - <i>Bundle size increase detected, please review</i>
  </td>
    </tr>
  </tbody>
</table>


Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
gestalt.js | :small_red_triangle: 0.1% | :small_red_triangle: 0.2% | 299.76 KB | 300.05 KB | 61.57 KB | 61.71 KB
gestalt.es.js | :small_red_triangle: 0.1% | :small_red_triangle: 0.3% | 283.46 KB | 283.75 KB | 60.63 KB | 60.79 KB
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by bradencanderson** [comment]
Can we call this `compose`?

File: packages/gestalt/src/icons/compose.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="#B5B5B5" d="M2.337 19.942l5.671-1.977L19.265 6.706c.981-.98.981-2.57 0-3.55l-1.42-1.421a2.51 2.51 0 0 0-3.55 0L3.036 12.992l-1.978 5.671a1.005 1.005 0 0 0 1.279 1.279M23 22c0 .55-.45 1-1 1H2c-.55 0-1-.45-1-1s.45-1 1-1h20c.55 0 1 .45 1 1"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/compose.svg]: just want to verify that this svg code went through @jaymarsh and was optimized?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
   </summary>
 
 ### Minor
-
+- Icon: Add compose icon (#352)
```
Comment (pfarejowicz) [inline CHANGELOG.md]: i just realized this is actually #358 not #352 , do you mind changing @jcpark89 https://github.com/pinterest/gestalt/pull/358/

File: packages/gestalt/src/icons/compose.svg (hunk 1 above)
Comment (pfarejowicz) [reply packages/gestalt/src/icons/compose.svg]: I just noticed the `fill` property is wrong here, I know we set colors using the props on the `Icon` component but do you mind updating this SVG source code to stay consistent with the rest of the icon set? You can look at the `pin` icon raw code as an example https://raw.githubusercontent.com/pinterest/gestalt/master/packages/gestalt/src/icons/pin.svg

**Pull request merged**